### PR TITLE
Update package

### PR DESCRIPTION
**Pull request by hunchr**
## Changes
- use lockfileVersion 3
- update devDependencies
- remove unused dependencies
- remove mocha because it wasn't used

### CODE REVIEW

File: assets.src/package-lock.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "redmine-tracky",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
Comment (sihu) [inline assets.src/package-lock.json:4]: what was the reason to delete the package.lock.json? the reason to have a package-lock is to freeze the dependency version and not break it randomly at a build. that's why we have that

File: assets.src/package-lock.json [hunk 2]
```diff
@@ -333,78 +359,72 @@
...
+      "version": "7.0.15",
+      "resolved": "https://registry.npmjs.org/@types/json-schema/-/json-schema-7.0.15.tgz",
+      "integrity": "sha512-5+fP8P8MFNC+AyZCDxrB2pkZFPGzqQWUzpSeuuVLvm8VMcorNYavBqoFcxK8bQz4Qsbn4oUEEem4wDLfcysGHA==",
+      "dev": true
     },
     "node_modules/@types/luxon": {
-      "version": "2.4.0",
-      "resolved": "https://registry.npmjs.org/@types/luxon/-/luxon-2.4.0.tgz",
-      "integrity": "sha512-oCavjEjRXuR6URJEtQm0eBdfsBiEcGBZbq21of8iGkeKxU1+1xgKuFPClaBZl2KB8ZZBSWlgk61tH6Mf+nvZVw==",
+      "version": "3.4.2",
```
Comment (sihu) [inline assets.src/package-lock.json:368]: the way to update packages should usually be to use `npm update`. like this you update everything - sometimes even major versions like here.

**Review by sihu — COMMENTED** [review]
have you tested the whole functionality? since you updated all npm packages there could be breaking changes etc?

**Comment by hunchr** [comment]
> have you tested the whole functionality? since you updated all npm packages there could be breaking changes etc?

yes, everything still works

**Pull request merged**